samples/CloudClipboard/CloudClipboard/CloudClipboard/Pages/Index.cshtml.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+#region v11 usings
```

Comment:
Why are we representing both v11 and v12?   Shouldn't we be highlighting the track two library?